build.gradle

```diff
@@ -91,6 +92,18 @@ jacocoTestReport {
   }
 }
 
+project.tasks['sonarqube'].dependsOn test, integration
+
+sonarqube {
+  properties {
+    property "sonar.projectName", "Reform :: spring-boot-template"
+    property "sonar.projectKey", "uk.gov.hmcts.reform.demo:spring-boot-template"
```

Comment:
Perhaps `demo` can be skipped here, don't you think?

Reply:
sure